rskj-core/src/main/java/co/rsk/peg/BridgeUtils.java

```diff
@@ -262,8 +265,18 @@ public static int countMissingSignatures(Context btcContext, BtcTransaction btcT
         TransactionInput input = btcTx.getInput(0);
         Script scriptSig = input.getScriptSig();
         List<ScriptChunk> chunks = scriptSig.getChunks();
+        Script redeemScript = new Script(chunks.get(chunks.size() - 1).data);
```

Comment:
This method seems to be used only for logging. We should make sure it is only called if logging level is enabled.

Reply:
This method = `countMissingSignatures`

Reply:
Done